### DISCUSSION TITLE
Stuck Deployment?

### DISCUSSION THREAD

**Discussion by marwan-at-work** [planetscale/discussion#48] [Bug Reports] [answered]
Hi there,

I have started a deployment request that has taken over 30 minutes and is not showing any sign of succeeding. Previous deployments have not taken more than 10 seconds. The only difference is that I'm dropping all of my tables in the latest/stuck 
deployment. 

I'd be happy to provide names/links to my org name and deployments for debugging on your end. 

Nonetheless, it would be great if deployments showed some form of indication of how long it might take or maybe some logs to see where it's at in the progress. 

Thanks!

**Comment by Anders-PlanetScale** [comment, marked as answer]
Howdy @marwan-at-work,
We have been experiencing some issues with deployments, I will open an internal issue so that our engineering team can work on debugging If you could create a private ticket here: https://www.planetscale.com/support where you list your DB name and branch name that would be helpful.

**Reply by marwan-at-work** [reply]
Done :)
